docs/faq.md

```diff
@@ -580,10 +622,11 @@ You can do that via the modifiers.ini file. Below is not a comprehensive list of
 No, Multitroid is a mod made by MilesTheNerd. While we greatly enjoy the mod and Miles is part of our team, it is not officially supported at present due to the time and energy required to provide official multiplayer support.
 
 #### How do I host on Multitroid?  
-1. You need to have the AM2RServer downloaded: <https://github.com/lassiterm/AM2R-Server/releases/latest>
-2. You need to either port forward, or use a VPN (like Hamachi, ZeroTier, Radmin etc.).
+1. Install [AM2RServer](https://github.com/lassiterm/AM2R-Server/releases/latest)
+2. Port forward the server IP, or use a VPN (like Hamachi, ZeroTier, Radmin etc.).
 3. Launch the AM2R Server.
 4. Give the other players your **external** IP address and port to connect with.
+
```

Comment:
This will break formatting. You probably want:
```
4. Give the other players your **external** IP address and port to connect with.  
    Port forwarding is different on every router, consult your routers manual or the internet for more info. For VPNs, consult their documentation. We cannot answer questions about specific routers or VPNs.
 ```

Reply:
Fixed